### PR TITLE
Text cleanup

### PR DESCRIPTION
**Pull request by HeatRiser**
Clean up a lot of text shown to users to clear up ambiguity
(also probably makes half of them sound worse because vars, will check those later)

### CODE REVIEW

File: addons/sourcemod/scripting/vsh/bosses/boss_seeldier.sp [hunk 1]
```diff
@@ -31,16 +31,16 @@ methodmap CSeeldier < SaxtonHaleBase
...
 		StrCat(sInfo, length, "\nMelee deals 124 damage");
 		StrCat(sInfo, length, "\nHealth: Low");
 		StrCat(sInfo, length, "\n ");
 		StrCat(sInfo, length, "\nAbilities");
 		StrCat(sInfo, length, "\n- Brave Jump");
 		StrCat(sInfo, length, "\n ");
 		StrCat(sInfo, length, "\nRage");
-		StrCat(sInfo, length, "\n- Summons 3 mini seeldiers");
-		StrCat(sInfo, length, "\n- 200%% Rage: Summons 6 mini seeldiers");
+		StrCat(sInfo, length, "\n- Summons 3 mini-SSeeldiers");
```
Comment (FortyTwoFortyTwo) [inline addons/sourcemod/scripting/vsh/bosses/boss_seeldier.sp:42]: Double S

**Review by FortyTwoFortyTwo — COMMENTED** [review]
current changes looks fine to me

File: addons/sourcemod/scripting/vsh/bosses/boss_seeldier.sp (hunk 1 above)
Comment (HeatRiser) [reply addons/sourcemod/scripting/vsh/bosses/boss_seeldier.sp:42]: why the double S? Thought it didn't have that

Comment (HeatRiser) [reply addons/sourcemod/scripting/vsh/bosses/boss_seeldier.sp]: nevermind i am a moron and is fixed now

**Review by FortyTwoFortyTwo — APPROVED** [review]

**Pull request merged**